src/vivliostyle/viewer.js

```diff
@@ -384,6 +384,12 @@ goog.scope(function() {
     };
 
     /**
+     * @return {Array<{width: number, height: number}>}
+     */
+    vivliostyle.viewer.Viewer.prototype.getPageSizes = function() {
+        return this.adaptViewer.pageSizes;
+    };
+    /**
```

Comment:
This public API need to be exported by `goog.exportProperty()` so that it is accessible when minified by Closure Compiler. See the following lines for examples.